proxy/src/main/java/org/apache/accumulo/proxy/Proxy.java

```diff
@@ -273,12 +248,15 @@ public static ServerAddress createProxyServer(HostAndPort address,
         }
 
         // Login via principal and keytab
-        final String kerberosPrincipal = properties.getProperty(KERBEROS_PRINCIPAL, ""),
-            kerberosKeytab = properties.getProperty(KERBEROS_KEYTAB, "");
+        final String kerberosPrincipal = clientProps
+            .getProperty(ClientProperty.AUTH_USERNAME.getKey(), "");
+        final String kerberosKeytab = clientProps
+            .getProperty(ClientProperty.AUTH_KERBEROS_KEYTAB_PATH.getKey(), "");
         if (StringUtils.isBlank(kerberosPrincipal) || StringUtils.isBlank(kerberosKeytab)) {
-          // ACCUMULO-3651 Changed level to error and added FATAL to message for slf4j capability
-          log.error("FATAL: Kerberos principal and keytab must be provided");
-          throw new RuntimeException();
+          String msg = String.format("Kerberos principal '%s' and keytab '%s' must be provided",
+              kerberosPrincipal, kerberosKeytab);
+          log.error(msg);
+          throw new RuntimeException(msg);
```

Comment:
An IllegalStateException might be more appropriate here. Also, it's bad practice to both log *AND* throw. We should pick one... either the caller is responsible for logging and handling the error, or we are. (I know this is existing code, but since you appear to be cleaning it up anyway... :smiley_cat: )

Reply:
I'll fix this